pytubefix/contrib/channel.py

```diff
@@ -456,15 +456,32 @@ def description(self) -> str:
         self.html_url = self.channel_url
         return self.initial_data['metadata']['channelMetadataRenderer']['description']
 
+    def find_videos_info(self, data):
+        """Recursively search for 'videos' in the text content of the JSON."""
+        if isinstance(data, dict):
+            for key, value in data.items():
+                if key == 'content' and isinstance(value, str) and 'videos' in value:
```

Comment:
@JuanBindez shouldn't this be 
````
and ' videos' in value:
````

Reply:
do the tests and if it's correct, make a PR